yarn-project/noir-protocol-circuits-types/src/index.ts

```diff
@@ -174,9 +199,13 @@ export async function executeReset<
     NUM_TRANSIENT_DATA_HINTS
   >,
   dimensions: PrivateKernelResetDimensions,
+  untrimmedPrivateKernelResetCircuitPrivateInputs?: PrivateKernelResetCircuitPrivateInputs,
```

Comment:
Can we add a comment for this arg? Because it's only here as a hack and we might be able to remove it in the future. If there's a test that has full data then we can generate the fixtures using trimmed inputs.